### DISCUSSION TITLE
开源商业化：满足各方底层需求 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#85] [Announcements]
# 开源商业化：满足各方底层需求 - 「开源之道」

https://opensourceway.community/posts/business_model/open-source-commercialization-to-meet-the-underlying-needs-of-all-parties/

**Comment by ZoomQuiet** [comment]
#是也乎
只能说, 老酒新瓶...

早在 如何成为hacker 中就已经指出过社区中不同贡献都是需要的现实;
而在 开源宣言中一开始就说明了商业化的希望,
甚至于在 GPL/自由软件许可证中, 就给出了合理的商业化方向;

也就是说, 开源从出生就包含丰富的商业化因素,
只是商业领域中的经理人们头脑比较固化, 一直没有发掘出开源经济的增长点儿而已;

以及商业成功, 一定不是底层需求, 
而是高层需求, 愿意溢价支付的;

否则平价出售, 没有增长点, 也就没有资本进入作大;

开源的归开源, 商业的归商业, 各得其所, 相互支撑就好;

并不存在只能商业的开源, 或是只能开源的商业...
